test/unit/tools/bigfiles.rb

```diff
@@ -4,12 +4,13 @@ module Tools
       # Test for the 'bigfiles' tool within the quality gem
       module BigFiles
         def bigfiles_expected_args
-          "--glob '{Rakefile,Dockerfile,{*,.*}.{rb,rake,gemspec," \
-          'swift,cpp,c,java,py,' \
-          'clj,cljs,scala,js,yml,sh,json},{src,app,config,db,lib,test,' \
-          'spec,feature}' \
-          '/**/{*,.*}.{rb,rake,gemspec,swift,cpp,c,java,py,clj,cljs,scala,js,' \
-          "yml,sh,json}}' " \
+          "--glob '{Dockerfile,Rakefile,{*,.*}.{c,clj,cljs,cpp,gemspec,java,js,json,py,rake,rb," \
+          'scala,sh,swift,' \
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.